Source/WebKitLegacy/mac/WebView/WebView.mm

```diff
@@ -820,6 +820,16 @@ void tryDestruction()
 
 } // namespace WebKit
 
+class LegacyHistoryItemClient : public WebCore::HistoryItemClient {
```

Comment:
I am confused as to why WebKit legacy needs both `GlobalHistoryItemClient` AND this class? What prevents us from using only one? How do they differ?

Reply:
Not r+'ing yet because of this.